### ISSUE TITLE
It seems that pre-training and fine-tuning are not aligned

### ISSUE THREAD

**Issue by Yuchen-Wang-CHN** [RManLuo/gfm-rag#22]
When I set the path of pre-traininged model in the stage2_qa_finetune.yaml, there will show an error "Missing key(s) in state_dict: "question_mlp.weight", "question_mlp.bias"". Pre-training use EntityNBFNet and fine-tuning use its subclass. So maybe the code in line 405, stage2_qa_finetune.py should be part loading of the model?

If I miss some part of the code, please tell me. Thanks.

**Comment by RManLuo** [comment]
Should be fixed at aeeb53d51f72323c773ef8aed10c13b3a363916d. Please have a try.

**Comment by xidian383** [comment]
@Yuchen-Wang-CHN @RManLuo 
Hello, may I ask if the loss eventually decreases during the pre-training phase? Looking forward to your reply!
